### PR TITLE
imp: add init-simapp script

### PR DESCRIPTION
**Pull request by julienrbrt**
## Description

Porting our script (https://github.com/cosmos/cosmos-sdk/pull/15846) in this repo as well.
I was very sad that I couldn't quickly setup simapp like I do in the SDK.
Please let me know if you have something like that somewhere else. I couldn't find it.

### Commit Message / Changelog Entry

```text
type: commit message
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/build/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

**Review by charleenfei — APPROVED** [review]
thanks for this! i think we generally try to run our qa/testing programatically via the e2e testing framework rather than manually, which might be the reason we didnt have this script in our repo

File: scripts/init-simapp.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+SIMD_BIN=${SIMD_BIN:=$(which simd 2>/dev/null)}
+
+if [ -z "$SIMD_BIN" ]; then echo "SIMD_BIN is not set. Make sure to run make install before"; exit 1; fi
+echo "using $SIMD_BIN"
+if [ -d "$($SIMD_BIN config home)" ]; then rm -r $($SIMD_BIN config home); fi
+$SIMD_BIN config set client chain-id demo
```
Comment (damiannolan) [inline scripts/init-simapp.sh:8]: maybe we should use a chain id that has a suffix incase things like ibc upgrades want to be tested.

```suggestion
$SIMD_BIN config set client chain-id simapp-1
```

File: scripts/init-simapp.sh [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+$SIMD_BIN config set client keyring-backend test
+$SIMD_BIN config set app api.enable true
+$SIMD_BIN keys add alice
+$SIMD_BIN keys add bob
+$SIMD_BIN init test --chain-id demo
```
Comment (damiannolan) [inline scripts/init-simapp.sh:13]:
```suggestion
$SIMD_BIN init test --chain-id simapp-1
```

File: scripts/init-simapp.sh [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+$SIMD_BIN genesis add-genesis-account alice 5000000000stake --keyring-backend test
+$SIMD_BIN genesis add-genesis-account bob 5000000000stake --keyring-backend test
+$SIMD_BIN genesis gentx alice 1000000stake --chain-id demo
```
Comment (damiannolan) [inline scripts/init-simapp.sh:16]:
```suggestion
$SIMD_BIN genesis gentx alice 1000000stake --chain-id simapp-1
```

**Review by damiannolan — APPROVED** [review]
Nice, thanks @julienrbrt! Added some suggestions on chain-id 

LGTM

**Review by DimitrisJim — APPROVED** [review]

**Pull request merged**